easybuild/easyconfigs/b/boost_histogram/boost_histogram-1.2.1-foss-2021a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonBundle'
+
+name = 'boost_histogram'
+version = '1.2.1'
+
+homepage = 'https://pypi.org/project/boost-histogram/'
```

Comment:
`https://boost-histogram.readthedocs.io` looks like a better homepage for `boost-histogram`?